### PR TITLE
2.0.0 release

### PR DESCRIPTION
**Pull request by frisbm**
# Description
2.0.0 Release that works with Neo4j 4.2.x as well as other updates for security reasons

## Motivation and Context

Project had gotten away from me for a while, making updates and cleaning some code preparing for a 2.0.0 release that works with Neo4j version 4.2.x

## How Has This Been Tested

Using the test containers, test harness in express and running through the demo

## Types of changes

- [x] Refactoring / dependency upgrade / docs change
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [x] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Non-functional requirement

## Visual screenshots

N/A

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -44,7 +44,7 @@ We recommend the use of Docker for running Neo4j locally to maintain static deve
 
 ### CyFHIR Commands
 
-Currently CyFHIR has 1 procedure and 1 aggregating function:
+Currently CyFHIR has 3 procedure2 and 2 aggregating functions:
```
Comment (DMadrid) [inline README.md:47]: typo

File: cyfhir/src/test/java/com/Optum/CyFHIR/functions/BundleTest.java [hunk 2]
```diff
@@ -29,10 +30,10 @@
 class BundleTest {
 
     @Container
-    private static final GenericContainer neo4j = new GenericContainer<>("neo4j:4.1.0")
-            .withEnv("NEO4J_AUTH", "neo4j/password")
+    private static final GenericContainer<?> neo4j = new GenericContainer<>(DockerImageName.parse("neo4j:4.2.7"))
+            .withExposedPorts(7687).withEnv("NEO4J_AUTH", "neo4j/password")
```
Comment (DMadrid) [inline cyfhir/src/test/java/com/Optum/CyFHIR/functions/BundleTest.java:34]: bump withEnv to next line for consistency

File: cyfhir/src/test/java/com/Optum/CyFHIR/functions/ResourceTest.java [hunk 3]
```diff
@@ -31,10 +32,10 @@
 class ResourceTest {
 
     @Container
-    private static final GenericContainer neo4j = new GenericContainer<>("neo4j:4.1.0")
-            .withEnv("NEO4J_AUTH", "neo4j/password")
+    private static final GenericContainer<?> neo4j = new GenericContainer<>(DockerImageName.parse("neo4j:4.2.7"))
+            .withExposedPorts(7687).withEnv("NEO4J_AUTH", "neo4j/password")
```
Comment (DMadrid) [inline cyfhir/src/test/java/com/Optum/CyFHIR/functions/ResourceTest.java:36]: same here

File: cyfhir/src/test/java/com/Optum/CyFHIR/procedures/BundleTest.java [hunk 4]
```diff
@@ -32,10 +33,10 @@
 class BundleTest {
 
     @Container
-    private static final GenericContainer neo4j = new GenericContainer<>("neo4j:4.1.0")
-            .withEnv("NEO4J_AUTH", "neo4j/password")
+    private static final GenericContainer<?> neo4j = new GenericContainer<>(DockerImageName.parse("neo4j:4.2.7"))
+            .withExposedPorts(7687).withEnv("NEO4J_AUTH", "neo4j/password")
```
Comment (DMadrid) [inline cyfhir/src/test/java/com/Optum/CyFHIR/procedures/BundleTest.java:37]: and here

**Review by DMadrid — DISMISSED** [review]

File: cyfhir/src/test/java/com/Optum/CyFHIR/functions/BundleTest.java (hunk 2 above)
Comment (MatthewFrisby) [reply cyfhir/src/test/java/com/Optum/CyFHIR/functions/BundleTest.java:34]: I actually had it bumped buy the mvn tdy package keeps moving it up

**Comment by MatthewFrisby** [comment]
@DMadrid You need to review it again since i pushed the typo fix

**Review by DMadrid — APPROVED** [review]

**Pull request merged**